app/src/main/java/com/example/rum8/fragments/PotentialRoommateProfileDefault.java

```diff
@@ -77,6 +78,7 @@ public void setUserProfileImage(Bitmap bitmap) {
 
     @Override
     public void goToProfileSettings() {
+        profilePicture.setImageDrawable(ContextCompat.getDrawable(getActivity(), R.drawable.images));
```

Comment:
Is this correct? This doesn't seem like code that goes in `goToProfileSettings`

Reply:
Yes, it is correct. It shows the picture for main page

Reply:
Wait, I'm wrong